utils/build/docker/python/django-poc.Dockerfile

```diff
@@ -12,6 +12,7 @@ COPY utils/build/docker/python/iast.py /app/iast.py
 ENV DD_TRACE_HEADER_TAGS='user-agent:http.request.headers.user-agent'
 ENV DD_REMOTECONFIG_POLL_SECONDS=1
 ENV _DD_APPSEC_DEDUPLICATION_ENABLED=false
+ENV DD_TRACE_API_VERSION=v0.4
```

Comment:
We should get an opinion from the system-test owners on this, not sure if it impacts (or may impact) other scenarios/features in the works

Reply:
Well, as all tracer are tested the same, it'd be better to set this at scenario level, to ensure that they are all configured the same. It's here : https://github.com/DataDog/system-tests/blob/main/utils/_context/_scenarios.py#L1076 in the params called `weblog_env`.

Reply:
Added those changes, thanks! @cbeauchesne